### PR TITLE
Add modules and TS defs for Volume, VolumeSlice, NRRDLoader

### PR DESCRIPTION
**Pull request by looeee**
Also moved `Volume` and `VolumeSlice` into the `misc/` folder.

### CODE REVIEW

**Comment by looeee** [comment]
Also added module for `libs/gunzip.min.js`. Just as with `inflate.min.js`, this is converted by hand since there is no equivalent ES6 module available.

File: examples/jsm/loaders/NRRDLoader.d.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+export class NRRDLoader {
+  constructor(manager?: LoadingManager);
+  manager: LoadingManager;
+  path: string;
+
+  fieldFunctions: object;
+
+  load(url: string, onLoad: (group: Volume) => void, onProgress?: (event: ProgressEvent) => void, onError?: (event: ErrorEvent) => void): void;
+  parse(data: string) : Volume;
+  parseChars(array: number[], start?: number, end?: number) : Volume;
```
Comment (Mugen87) [inline examples/jsm/loaders/NRRDLoader.d.ts]: `parseChars()` returns a string, not a volume.

File: examples/jsm/misc/VolumeSlice.d.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import {
+  Mesh,
+} from '../../../src/Three';
+
+import { volume } from './Volume';
+import { Matrix3 } from '../../../build/three.module';
```
Comment (Mugen87) [inline examples/jsm/misc/VolumeSlice.d.ts]: This looks strange. You can put `Matrix3` into the first import statement at the top instead. So:
```ts
import {
  Matrix3,
  Mesh
} from '../../../src/Three';
```

Comment (looeee) [reply examples/jsm/misc/VolumeSlice.d.ts]: Oops, I missed that. It was the VScode auto import function. Mostly it gets things right, sometimes... Not so much. 

I'll fix that in the morning.

**Comment by looeee** [comment]
@Mugen87 this should be ready to merge 👍

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**